### ISSUE TITLE
Can I use cereal to Deserialize XML to C++

### ISSUE THREAD

**Issue by tamndse62381** [USCiLab/cereal#605]
I'm writing a CMake project to Deserialize an XML schema (.xsd) to an C++ class. I have read the document and most of them are covered with Serialization. I hope someone can help me to find out if cereal can use for Deserialize or not? If yes which part of the document is used for it? Thanks in advance, I'm a newbie.

**Comment by AzothAmmo** [comment]
If you are trying to generate code from a schema, that is not possible with cereal. cereal is designed for data serialization only.

**Comment by tamndse62381** [comment]
> If you are trying to generate code from a schema, that is not possible with cereal. cereal is designed for data serialization only.

Yeah, I have already figured it out and if there is can you tell me which tools/app/library that can generate code from schema? It's like a JAXB(a library within Java) or xsd.exe for C# code generation,
